.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,33 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.187.0/containers/rust
+{
+	"name": "Rust",
+    "dockerComposeFile": ["../docker-compose.yaml"],
+    "service": "rust-client",
+	"workspaceFolder": "/workspace/rust-socketio",
+    "shutdownAction": "stopCompose",
```

Comment:
Could you please reformat the `.devcontainer.json` file? The formatting looks a bit off.

Reply:
fixed in https://github.com/1c3t3a/rust-socketio/pull/50/commits/26d44c2358365bcfe3add1a7ece4a860eb4ae24b